content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,43 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
+[Sensu](https://sensu.io) is an open source monitoring and observability pipeline with a commercial distribution which offers additional features for scalability. The entire Sensu observabliity pipeline configuration can be defined as code (declarative JSON or YAML template files). It can reuse existing Nagios plugins.
 
 ### Scope
 
-The same general scope differences as in the case of [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
-
-There is also a [client socket](https://docs.sensu.io/sensu-core/latest/reference/clients/#what-is-the-sensu-client-socket) permitting ad-hoc check results to be pushed into Sensu. 
+As an observability _pipeline_, Sensu focuses on processing and alerting of observability data as a stream of [Events](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/). It handles deduplication, correlation, and other filtering logic and then processes events by sending alerts, creating and resolving tickets in incident management systems, storing event & telemetry data in any number of data platforms, and managing automated remediation workflows. Sensu also provides a lightweight [monitoring agent](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/) (event producer) for monitoring of ephemeral cloud compute and distributed systems.
 
 ### Data model
 
-Sensu has the same rough data model as [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios).
+In Sensu, events (which can include service health and/or [metric](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metric-attributes) data) are identified by an [entity](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-entities/entities/) name (e.g. server, cloud compute instance, container, or service), an event name, and optional [key-value metadata](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metadata-attributes) called "labels" or "annotations".
```

Comment:
Great point. I linked to the [`metrics`](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metric-attributes) specification documentation but didn't actually explain how Sensu represents metric [`points`](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#points-attributes). I'll update that now.

To be clear, because Sensu is an event-based system, the "data model" is oriented around events, some of which may optionally contain metrics. The updated description will explain this better: 

> In Sensu, events (which can include service health and/or [metric](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metric-attributes) data) are identified by an [entity](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-entities/entities/) name (e.g. server, cloud compute instance, container, or service), an event name, and optional [key-value metadata](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metadata-attributes) called "labels" or "annotations". The Sensu Event payload may include one or more metric [`points`](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#points-attributes), represented as a JSON object containing a `name`, `tags`, `timestamp`, and `value` (always represented as a float).

I won't add an example in the PR, but the Sensu event payload itself contains a number of numerical data type fields (not just strings), specifically including the above-mentioned `metrics` field; here's a pseudo event example: 

```json
{
  "check": {},
  "entity": {},
  "metrics": {
    "handlers": [
      "influxdb"
    ],
    "points": [
      {
        "name": "sensu-go-sandbox.curl_timings.time_total",
        "tags": [
          {
            "name": "region",
            "value": "us-west-1"
          }
        ],
        "timestamp": 1552506033,
        "value": 0.005
      },
      {
        "name": "sensu-go-sandbox.curl_timings.time_namelookup",
        "tags": [
          {
            "name": "region",
            "value": "us-west-1"
          }
        ],
        "timestamp": 1552506033,
        "value": 0.004
      }
    ]
  },
  "metadata": {
    "name": "example",
    "namespace": "example",
    "labels": {},
    "annotations": {}
  },
  "timestamp": 1552506033,
}
```

Reply:
The data model here isn't just about metrics. If the event payload is basically an arbitrary set of structured data, you can say that.